### DISCUSSION TITLE
[Question] Gen 1/2 to 3+ converting

### DISCUSSION THREAD

**Discussion by danimesq** [kwsch/PKHeX#3551] [General]
Is PKHeX's method updated to be the same as used in the 3DS' Virtual Console?
@kwsch

**Comment by sora10pls** [comment]
PKHeX simulates official transfer methods as best as it possibly can. As such, no official means of transfer for Generations 1/2 to 3 exist, so these are not legal. You can enable incompatible transfers in your PKHeX settings, but the end results are still illegal.

PKHeX can, though, transfer PK1s/PK2s to PK7s onward, as you're able to transfer from the Virtual Console titles to Pokémon Bank. Example below is the code that converts a PK1 to a PK7.
https://github.com/kwsch/PKHeX/blob/038722ee09fa1139b6492056e9ca359d2712f4d2/PKHeX.Core/PKM/PK1.cs#L161-L241

**Comment by danimesq** [comment]
Thanks, @sora10pls!

So, Pokemon X/Y were never updated to support the Pokemon Bank (but only the Poke Transporter from BW/BW2) and only SM/USUM?

Also, would it be legal to copy from Gen7 back to Gen3, train these pokemons, then copy their updated stats/evolution back to Gen7?

**Comment by sora10pls** [comment]
Correct, Virtual Console Pokémon can only be sent to games that are Generation 7 or higher.

No. Backwards transfers are not possible (sans Time Capsule in the GB era games), so a Pokémon that exists in Gen 7 can never legally move back to Gen 1-6 games. They can only move up.

**Comment by danimesq** [comment]
Right. So I'll have to be patient and some Gen2 Pokemons reserved for Sinnoh (Such as Gligar), I'll have to use the Switch's ShiningPearl instead of the DS' Diamond.

Last (and off-topic/controverse; can't be discussed about legality) question.
I'm developing a recorded PokeJourney standard with some rules that slip from Pokemon's official ones; on this standard, it is permitted to transfer Pokeballs from a generation to another (gen 1 onwards).
So, I've started Pokemon Ruby with pokeballs (from Pokemon Crystal/Johto) and caught Prof. Birtch's foe Poochyena. I'd became attached to Nina but PKHeX says it is invalid because of PID/Location mismatch. What does this error means and, is it possible to fix it (legality about this method is apart)?

**Comment by danimesq** [comment]
And back at discussing legality - also off-topic:
Is it okay to use the same TID/SID from a game to another?

**Reply by Atrius97** [reply]
Generally no. The chances of that occurring naturally are astronomically tiny (as in "the sun would burn out before it happens by accident"), and it also causes legality issues when a Pokémon is from a different game/generation, but also is not considered an "outsider Pokémon". That would allow some values that would otherwise be impossible, to exist.
